ubuntu_install/install_openssl.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+set -e
+
+# "$(uname -s)-$(arch)"
+OPENSSL_ARCH="linux-x86_64"
```

Comment:
NOTE: This part is specific to a machine. It seems to really only matter since OS X [is a problem][1] and has to use `./Configure` instead of `./config`.

[1]: https://wiki.openssl.org/index.php/Compilation_and_Installation#Configure_.26_Config